engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-traces-modal.controller.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.content-hosts.controller:ContentHostsBulkTracesController
+ *
+ * @requires $scope
+ * @requires $state
+ * @requires $timeout
+ * @requires $window
+ * @requires $uibModalInstance
+ * @requires HostBulkAction
+ * @requires Organization
+ * @requires CurrentOrganization
+ * @requires Notification
+ * @requires Nutupane
+ * @requires BastionConfig
+ * @requires HostTraces
+ * @requires hostIds
+ *
+ * @description
+ *   Provides the functionality for the content host package list and actions.
+ */
+/*jshint camelcase:false*/
+angular.module('Bastion.content-hosts').controller('ContentHostsBulkTracesController',
+    ['$scope', '$state', '$timeout', '$window', '$uibModalInstance', 'HostBulkAction', 'Organization', 'CurrentOrganization', 'Notification', 'Nutupane', 'BastionConfig', 'HostTraces', 'hostIds',
+    function ($scope, $state, $timeout, $window, $uibModalInstance, HostBulkAction, Organization, CurrentOrganization, Notification, Nutupane, BastionConfig, HostTraces, hostIds) {
+        var tracesNutupane;
+
+        tracesNutupane = new Nutupane(HostBulkAction, hostIds, 'hostTraces');
```

Comment:
Unless there's some Angular convention I don't know about, it makes more sense to me to declare and assign this on the same line:
```js
var tracesNutupane = new Nutupane(HostBulkAction, hostIds, 'hostTraces');
```

Reply:
Fixed